core/api/views.py

```
@@ -57,4 +58,220 @@
             },
             status=status.HTTP_200_OK
         )
+class UserContributionView(APIView):
+    """
+    Get all contributions that user have created.
+    user must be authenticated to view their uploaded contribution
+
+    """
+    permission_classes = [IsAuthenticated]
```

Ruff (0.8.2)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)